proto/messages.proto

```diff
@@ -225,6 +229,8 @@ message Asset {
     RemoteData uploaded = 4;
   }
   optional Preview preview = 5;
+  optional bool expects_read_confirmation = 6 [default = false]; // whether the sender is expecting to receive a read confirmation
```

Comment:
I'm not sure if this is the right level to put the flag. Should we put it in `Original` instead? Is there ever a case where `Original` is not sent?

Reply:
I think this is good place. All supported message types have this flag at the root level.